### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by kalininskiy** [jaedb/Iris#348]
## Expected behavior
install with
pip install Mopidy-Iris

## Actual behavior
not install with error
pylast requires Python '>=3.5' but the running Python is 2.7.13

## Steps to replicate
1. pip install Mopidy-Iris
2.
```
pi@smartrpi:~ $ pip install mopidy-iris
Collecting mopidy-iris
  Using cached https://www.piwheels.org/simple/mopidy-iris/Mopidy_Iris-3.31.8-py2.py3-none-any.whl
Collecting tornado<5.0,>=3.2 (from mopidy-iris)
Collecting setuptools>=3.3 (from mopidy-iris)
  Using cached https://files.pythonhosted.org/packages/37/06/754589caf971b0d2d48f151c2586f62902d93dc908e2fd9b9b9f6aa3c9dd/setuptools-40.6.3-py2.py3-none-any.whl
Collecting ConfigObj>=5.0.6 (from mopidy-iris)
  Using cached https://files.pythonhosted.org/packages/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz
Collecting requests>=2.0.0 (from mopidy-iris)
  Using cached https://files.pythonhosted.org/packages/7d/e3/20f3d364d6c8e5d2353c72a67778eb189176f08e873c9900e10c0287b84b/requests-2.21.0-py2.py3-none-any.whl
Collecting Mopidy>=2.0 (from mopidy-iris)
  Using cached https://files.pythonhosted.org/packages/01/6f/bd3df70638fa740edfe2913eecce952712515487a7d70bf8fe3e0a753576/Mopidy-2.2.2-py2.py3-none-any.whl
Collecting Mopidy-Local-Images>=1.0 (from mopidy-iris)
  Using cached https://files.pythonhosted.org/packages/0d/76/e49b06e6d2a7aa4b94a407ab400e138acbd0e01791adfdc5492084da5411/Mopidy_Local_Images-1.0.0-py2.py3-none-any.whl
Collecting pylast>=1.6.0 (from mopidy-iris)
  Using cached https://www.piwheels.org/simple/pylast/pylast-3.0.0-py2.py3-none-any.whl
pylast requires Python '>=3.5' but the running Python is 2.7.13
```

#### Operating systems affected
- [ ] Windows:
- [ ] Mac: 
- [ ] Linux: 
- [ ] iOS: 
- [ ] Andriod:
- [x] Other: Raspbian

#### Browsers affected
- [ ] Chrome: 
- [ ] Firefox: 
- [ ] Internet Explorer:
- [ ] Opera: 
- [ ] Other:

#### Screen sizes affected
- [ ] Huge (ie desktop)
- [ ] Large (ie laptop)
- [ ] Medium (ie slim laptop/tablet)
- [ ] Small (phone)

## Before you submit
- [x] Yes, I have tested this issue thoroughly
- [ ] Yes, I have attached screenshots and/or screencast of the issue in action
- [x] Yes, I have included any relevant debug data and error logs

**Comment by jaedb** [comment]
This is likely to be because you're running `pip` on Python3. Are you able to install pip for Python2.7 and try again? Related: #347

**Comment by jaedb** [comment]
I actually think we may be safe to remove `pylast` completely. Watch this space...

**Comment by jaedb** [comment]
Resolved in #349
